src/sql/azdata.d.ts

```diff
@@ -3434,7 +3434,10 @@ declare module 'azdata' {
 	}
 
 	export interface TextComponentProperties extends ComponentProperties, TitledComponentProperties {
-		value?: string | undefined;
+		/**
+		 * Provide value to be displayed in the text component. An array of value will be displayed as an unordered list.
```

Comment:
as an unordered list? it could also be ordered list, right?

Reply:
Need to add a better comment over there. By default it will be an unordered list.